src/main/scala/nl.knaw.dans.easy.validatebag/BagStore.scala

```diff
@@ -47,7 +47,11 @@ trait BagStore extends DebugEnhancedLogging {
       .header("Accept", "text/plain")
       .timeout(connTimeoutMs = connectionTimeoutMs, readTimeoutMs = readTimeoutMs)
       .method("HEAD")
-      .asBytes.code == 200
+      .asBytes.code match {
+      case 200 => true
+      case 400 => false
```

Comment:
I'm confused, 404 is not found, 400 is a bad request

Reply:
This fix restores the original behavior of my manual command line test with a fake is-version-of in bag-it.txt in an otherwise valid bag.